### DISCUSSION TITLE
求游戏龙之信条

### DISCUSSION THREAD

**Discussion by LIUCG92** [Gamuxorg/bbs#202] [投递和请求游戏]
游戏名称：龙之信条
游戏名称(英文)：Dragon's Dogma
下载地址 :（如果有的话）：https://store.steampowered.com/app/367500/Dragons_Dogma_Dark_Arisen/?l=tchinese&curator_clanid=28673811

**Comment by shouhuanxiaoji** [comment]
这个没有linux原生，你之前是玩的wine？

**Comment by LIUCG92** [comment]
我还没玩过这个游戏，一直想玩可惜电脑不是windows。不过看Proton上显示是金色，以为可以像老头环一样顺利运行

**Comment by shouhuanxiaoji** [comment]
好的，会尝试一下

**Comment by LIUCG92** [comment]
> 好的，会尝试一下

多谢，麻烦了
